CMakeLists.txt

```diff
@@ -44,20 +44,8 @@ find_package(Geant4 REQUIRED)
 # Minimum version is 1.2.8 due to the use of VecGeom::Logger
 find_package(VecGeom 1.2.8 REQUIRED)
 
-# Celeritas *might* be coming from a superproject...
-if(G4VG_BUILD_TESTS)
-  if(NOT Celeritas_FOUND)
-    find_package(Celeritas QUIET)
-    if(NOT Celeritas_FOUND)
-      add_subdirectory(external)
-    endif()
-  endif()
-endif()
-
-if(NOT Celeritas_FOUND)
-  # Load CUDA RDC utils from VecGeom
-  include(CudaRdcUtils)
-endif()
+# Load CUDA RDC utils from VecGeom
+include(CudaRdcUtils)
```

Comment:
The only gotcha here is that VecGeom only supplies this from `v1.2.10`, so should we bump the `find_package` min version above as well?

Reply:
We're currently distributing CudaRdcUtils with G4VG as well, so we're not relying on pulling it from Celeritas. The `if` was only needed because we were loading the celeritas external before this include here.

Reply:
Ah, o.k. I missed the file being included here!